blueocean-dashboard/src/main/resources/jenkins/plugins/blueocean/dashboard/Messages_de.properties

```diff
@@ -0,0 +1,64 @@
+Branches=Ableger
+Build=Build
+Changes.by=\u00C4nderungen von {0}
+Changes=\u00C4nderungen
+Changes.condense={0} \u00E4nderungen
+Commit=Commit
+Completed=Komplett
+Dashboard=Instrumententafel
+Date=Datum
+Date.readable.short=MMM DD h:mma Z
+Date.readable.long=MMM DD YYYY h:mma Z
+Date.duration.format=m[ Minuten] s[ Sekunden]
+Date.duration.hint.format=M [Monate], d [Tage], h[Std.], m[m], s[s]
+Download.artifact=Herunterladen des Artefaktes
+Duration=Dauer
+Duration.param=Dauer {0}
+# the emptyState keys assume that you will parse them with a markdown parser
+EmptyState.activity=# Auf die Pl\u00E4tze, fertig...\nHmm, es liegen keine Ausf\u00fchrungen f\u00FCr vor.\n\nF\u00FCgen sie ein Commit zum Repository _{0}_ oder starten Sie die Ausf\u00fchrung per Hand.
+EmptyState.artifacts=Es liegen keine Artefakte diese Ausf\u00fchrung vor.\n\n
+EmptyState.branches=# Ableger ziehen\nErstellen Sie einen Ableger (Branch) in Ihrem Repository _{0}_ und Jenkins wird Ihre \u00C4nderungen testen.\n\nProbieren Sie es und werden Sie der/die Held(in) Ihres Teams.
+EmptyState.branches.notSupported=# Ableger nicht unterst\u00FCtzt.\nAbleger (Branch) builds funktionieren nur mit dem _Multibranch Pipeline_ Aufgabentyp. Dies ist nur ein guter Grund Jenkins Pipeline zu nutzen.\n\n[Weitere gute Gr\u00FCnde](https://jenkins.io/doc/book/pipeline-as-code/)
+EmptyState.changes=Es liegen keine \u00C4nderungen f\u00FCr diese Ausf\u00fchrung vor.\n\n
+EmptyState.pr.notSupported=# Integrationsanfrage f\u00FCr \u00C4nderungen (Pull Requests) nicht unterst\u00FCtzt.\nDie Validierung von Integrationsanfragen funktionieren nur mit dem _Multibranch Pipeline_ Aufgabentyp. Dies ist nur ein guter Grund Jenkins Pipeline zu nutzen.\n\n[Weitere gute Gr\u00FCnde](https://jenkins.io/doc/book/pipeline-as-code/)
+EmptyState.pr=# Push me, pull you\nWenn eine Integrationsanfrage in Ihrem Repository _{0}_ erstellt wird, wird Jenkins diese testen und den Status der Integrationsanfrage Github melden.
+EmptyState.tests=Es liegen keine Tests f\u00FCr diese Ausf\u00fchrung vor.\n\n
+Enable=Aktivieren
+Failed=Fehlgeschlagen {0}
+Failing={0} fehlgeschlagen
+Failures=Fehler
+Fixed=Fixiert
+Health=Gesundheit
+Latest.build=Letzte Ausf\u00fchrung
+Latest.message=Letzte Nachricht
+Loading=Laden...
+Message=Nachricht
+More=Mehr anzeigen
+Name=Name
+New.failures=Neue Fehler
+New.Pipeline=Neue Pipeline
+No.changes=Keine \u00C4nderungen
+noSteps=Es gibt keine Schritte.
+Not.found.heading=Seite nicht gefunden (404)
+Not.found.message=Jenkins konnte die Seite die Sie suchen nicht finden. Bitte kontrollieren Sie die URL oder dr\u00FCcken Sie den zur\u00FCck Knopf.
+Open.dashboard=Instrumententafel anzeigen
+Passed=Erfolgreich {0}
+Passing={0} erfolgreich
+Passing.singular=Erfolgreich
+Pipeline=Pipeline
+PR=Integrationsanfrage
+Run.now=Ausf\u00fchrung starten
+Serenity=Wie schaut es?
+Show.complete.logs=Ganze Protokoll
+Size=Gr\u00F6sse
+Skipped.singular=\u00DCbersprungen
+Skipped=\u00DCbersprungen {0}
```

Comment:
siehe oben ist ATM in einem hidden block
![screenshot from 2016-10-25 12-37-04](https://cloud.githubusercontent.com/assets/596701/19682926/2754877e-9ab0-11e6-91f1-0e3d1e299cbe.png)